Can't install it using Cordova@8.1.2

I tried install it in my project following README instructions but I'm stuck on this

**> cordova plugin add https://github.com/rtk/cordova-cookie-emperor.git --save**
Failed to fetch plugin https://github.com/rtk/cordova-cookie-emperor.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received type undefined
[ERROR] An error occurred while running subprocess cordova.

Searching for any leads that could help me I've found this;

https://ourcodeworld.com/articles/read/455/cordova-error-failed-to-fetch-plugin-npm-not-being-able-to-find-a-package-json-in-a-package-you-are-trying-to-install

>you will need to create a new package.json file in the root folder of your plugin.

--- a/package.json
+++ b/package.json
@@ -22,5 +22,9 @@
     "cordova-ios"
   ],
   "author": "Rouven Kruse",
-  "license": "MIT"
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/RTK/cordova-cookie-emperor/issues"
+  },
+  "homepage": "https://github.com/RTK/cordova-cookie-emperor#readme"
 }

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/RTK/cordova-cookie-emperor"
+    "url": "git+https://github.com/RTK/cordova-cookie-emperor"
   },
   "keywords": [
     "cordova",

